roles/radicale/tasks/radicale.yml

```diff
@@ -0,0 +1,40 @@
+- name: Install radicale dependencies
+  apt: pkg={{ item }} state=present
+  with_items:
+    - apache2-utils
+  tags:
+    - dependencies
+
+- name: Install radicale
+  apt: pkg=radicale update_cache=yes
```

Comment:
This package is from official repository, I think update cache is already done in common role